### PR TITLE
Polyhedron_demo : Update the ids when the item's geometry is changed

### LINKED ISSUE #1706: IDs (displayed with CTRL+I) are not refreshed after an Euler operations
I have tried to display the IDs, using <kbd>Ctrl+I</kbd>, and at the same time use Euler operations on the polyhedron, to add a face to the border. After the Euler operation succeeded, the IDs were not refreshed.

### PR DESCRIPTION
**Pull request by maxGimeno**
This PR fixes #1706 :

 Recomputes the ids in Scene_polyhedron_item::invalidateOpenGLBuffers()

### CODE REVIEW

File: Polyhedron/demo/Polyhedron/Scene.cpp [hunk 1]
```diff
@@ -1158,6 +1158,19 @@ void Scene::printPrimitiveIds(CGAL::Three::Viewer_interface* viewer)
...
+void Scene::updatePrimitiveIds(CGAL::Three::Viewer_interface* viewer, CGAL::Three::Scene_item* it)
+{
+  if(it)
+  {
+    //Only call printPrimitiveIds if the item is a Scene_print_interface_item
+    Scene_print_interface_item* item= dynamic_cast<Scene_print_interface_item*>(it);
+    if(item)
+    {
+      item->printPrimitiveIds(viewer);
+       item->printPrimitiveIds(viewer);
```
Comment (lrineau) [inline Polyhedron/demo/Polyhedron/Scene.cpp]: Why is that called twice?!

Comment (maxGimeno) [inline Polyhedron/demo/Polyhedron/Scene.cpp]: The first one hides the ids, the second one shows them again. It is the second one that recomputes the ids.

Comment (lrineau) [inline Polyhedron/demo/Polyhedron/Scene.cpp]: ... Would you mind writing a comment, there, to explain that fact? :-)

And also fix the indentation.

Comment (maxGimeno) [inline Polyhedron/demo/Polyhedron/Scene.cpp]: Of course

**Comment by lrineau** [comment]
Maxime, there is a conflict between `master` and this PR: `master` added `setOffset()` and this PR added `updatePrimitiveIds()`.

I think you can resolve the conflict using the Github web interface, with the button "Resolve conflicts" below.

File: Polyhedron/demo/Polyhedron/Viewer.cpp [hunk 2]
```diff
@@ -1781,5 +1781,13 @@ qglviewer::Vec Viewer::offset()const { return d->offset; }
 void Viewer::setSceneBoundingBox(const qglviewer::Vec &min, const qglviewer::Vec &max)
 {
   QGLViewer::setSceneBoundingBox(min+d->offset, max+d->offset);
+
+void Viewer::updateIds(CGAL::Three::Scene_item * item)
+{
```
Comment (lrineau) [inline Polyhedron/demo/Polyhedron/Viewer.cpp]: Syntax error! Please check your modification before pushing a pull-request!

**Pull request merged**